ui_tests/src/test/resources/com/saucedemo/account/signIn.feature

```diff
@@ -11,7 +11,23 @@ Feature: Sign In
   @signIn
   Scenario: Sign in with invalid password
     Given I open start page
-    When I set 'standard_user' as username on Sign in page
-    And I set 'thisIsSomeInvalidPassword' as password on Sign in page
-    And I click 'Sign In' button on Sign in page expecting failure
-    Then Error message 'Epic sadface: Username and password do not match any user in this service' is available on Sign in page
+    When I set 'problem_user' as username on Sign in page
+    And I set 'secret_sauce' as password on Sign in page
+    And I click 'Sign In' button on Sign in page
+    Then I am on Products page
+
+
+  @signIn
+  Scenario: Valid user sign in
+    Given I open start page
+    When I set 'performance_glitch_user' as username on Sign in page
+    And I set 'secret_sauce' as password on Sign in page
+    And I click 'Sign In' button on Sign in page
+    Then I am on Products page
+
+
+
+
+
+
+
```

Comment:
OK UNDERSTOOD DONE BY MISTAKE, IS IT ALRIGHT NOW ?

Reply:
You need to undo them. In order to do it, you will need to compare your version of this file with the one in master branch (I did show during the class: Right click on the file > git > compare with branch > master), revert the changes on the comparison popup so that content would become equal and then commit (and push) the updated file.  
Here's a video of the steps performed (without commit and push): http://somup.com/cYeUFD1FJ5